decidim-vocdoni/decidim-module-vocdoni
# Fix admin preview mode and enhance verification logic

- [x] Census check fix: The verification logic was updated to handle cases where users entered incorrect token to vote.
- [x] Admin preview mode fix: Fixed the preview mode logic to allow administrators to access the preview regardless of their inclusion in the census. 

**Tests:**

- Create an election with external census, run. Go to voting, enter incorrect token details, the message "The email or token is not valid. Please check these details and try again."
<img src="https://github.com/decidim-vocdoni/decidim-module-vocdoni/assets/60363870/b0fe01df-3a5d-4cb8-b58d-3e00ce88e0ad" width="600">

- Create an election with external census, do not run. Go to the public page of the election, as admin, open preview mode. Preview mode should open a page with login and token form, then allow voting.
- Create an election with internal census, do not run it. Go to the public page of the election, as admin, open preview mode. Preview mode should open the page with the first question and allow voting.
